Lesson 23 Gokhan/styles.css

```diff
@@ -0,0 +1,78 @@
+* {
+    margin: 0;
```

Comment:
It's usually not good (and not efficient) to remove margin and padding from all elements at once. Better to remove margin and padding selectively from elements that really don't need it (e.g. body, ul, ol, etc.)